app/src/main/java/at/roteskreuz/stopcorona/screens/menu/MenuController.kt

```diff
@@ -127,12 +148,18 @@ class MenuController(
                 .addTo(modelList)
         }
 
-        if (ownHealthStatus is HealthStatusData.SicknessCertificate) {
+        val isRedRevokingEnabled = dateOfFirstMedicalConfirmation
```

Comment:
The same logic regarding revoking period as above should apply here as well.

Reply:
Updated.